utils/statediff/src/lib.rs

```diff
@@ -158,6 +160,133 @@ pub struct MinerState {
     pub deadline_cron_active: bool,
 }
 
+#[derive(Clone, Default, Serialize_tuple, Deserialize_tuple, Debug)]
```

Comment:
To my understanding, this won't (and should not) be used outside the statediff. How about making those private or `pub(crate)` so they won't play tricks on others? In other words, I expect my completion to suggest me to import e.g. the `MarketState` from `builtin-actors` and not from here.

Reply:
Good point, I'll fix up the access throughout the crate to be sure